Kudu.Services/ServiceHookHandlers/OneDriveHandler.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Web;
+using Kudu.Contracts.Settings;
+using Kudu.Contracts.Tracing;
+using Kudu.Core;
+using Kudu.Core.Deployment;
+using Kudu.Core.SourceControl;
+using Kudu.Services.FetchHelpers;
+using Newtonsoft.Json.Linq;
+
+namespace Kudu.Services.ServiceHookHandlers
+{
+    public class OneDriveHandler : IServiceHookHandler
+    {
+        private readonly OneDriveHelper _oneDriveHelper;
+
+        public OneDriveHandler(ITracer tracer,
+                              IDeploymentSettingsManager settings,
+                              IEnvironment environment)
+        {
+            _oneDriveHelper = new OneDriveHelper(tracer, settings, environment);
+        }
+
+        public DeployAction TryParseDeploymentInfo(HttpRequestBase request, JObject payload, string targetBranch, out DeploymentInfo deploymentInfo)
+        {
+            deploymentInfo = null;
+            if (!string.Equals("OneDrive", payload.Value<string>("deployer"), StringComparison.OrdinalIgnoreCase))
```

Comment:
deployer is redundant with Url, no?   For github, bitbucket or dropbox, we don't need this, correct?